README.md

```diff
@@ -37,6 +37,62 @@ Follow installation instructions [here](https://pygraphviz.github.io/documentati
 
 ## Examples
 
+### Knowledge graph construction
+
+```python
+# Instantiate Entity and Relation objects
+entities = [
```

Comment:
Now the entity and relation definition becomes the longest part, we need to find shortcuts for this definition :) but not  now :)

Reply:
If the goal is to make more user-friendly, perhaps a less verbose way of defining this could be a list of tuples:

```python
entities = [
    ("PERSON", "An individual human being"),
    ("ORGANIZATION", "A structured group of people with a common purpose."),
    ...
]
```

Reply:
I'd say in the minimal approach, the description could also be skipped - when the meaning is obvious, there is no need to have it. If people need to add a description, then they can switch to the verbose SchemaEntity way.

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/165/commits/bf5e638f0220db9ba980d5811fd7f5e85560b395

Reply:
We can update the example in the README, right?